telemetry-http-okhttp/README.md

```diff
@@ -2,9 +2,6 @@ This module contains an implementation of the required http client interface for
 
 * `OkHttpPoster` : Implements the `HttpPoster` interface, using okhttp.
 
-In addition, it has two builder shims for creating telemetry senders that use okhttp: 
-`com.newrelic.telemetry.SimpleMetricBatchSender` and `com.newrelic.telemetry.SimpleSpanBatchSender`
-
```

Comment:
this comment should probably still be here until we can revisit the `Simple*` stuff